sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1,1320 +1,1365 @@
+|===
+[[sharepoint-onpremise_security-token]]
+===== Security Token
 
-[[sharepoint-onpremise_kerberos]]
-=== Kerberos
 
-==== Parameters
+Deprecated the SecurityTokenConnectionStrategy connection provider because the BasicConnectionProvider already obtains the security token
 
+
+====== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String | Username that you want to login with. Domain should not be included. | |x
-| Password a| String | Password for the user that you are going to login. | |x
-| Site Url a| String | URL for the SharePoint instance that the connector is going to interact with. | |x
-| SPN a| String | (Optional) SPN for the SharePoint Web Service. | |
-| Realm a| String | (Optional) The case-sensitive default realm (Domain Name) that the user belongs to. | |
-| KDC a| String | (Optional) The KDC (usually the Domain Controller name) to authenticate the user. | |
-| Login Properties File Path a| String | (Optional) Path to a customized Login Properties File. When not specified, default values which usually work for most cases are set up. | |
-| Kerberos Properties File Path a| String | (Optional) Path to a customized Kerberos Properties File. | |
-| Disable SSL certificate validation a| Boolean | When dealing with HTTPS certificates, if the certificate is not signed by a trusted partner, the server might respond with an Exception. To prevent this, it is possible to disable the CN (Common Name) check.  Note: This is not recommended for production environments. | false |
-| Proxy Settings a| One of:
-
-* proxy
-* ntlm-proxy | Settings for the proxy server that the connector is going to send data through. | |
-| Reconnection a| Reconnection | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | |
+| Security Token a| String |  |  | x
+| Security Token Provider a| <<SecurityTokenProvider>> |  (Optional) Provider instance that provides the security token when required. |  |
+| Site Url a| String |  URL for the target SharePoint instance |  | x
+| Disable SSL certificate validation a| Boolean |  Disables the CN (Common Name) check so that the server won't throw an exception if the HTTPS certificate is not signed by a trusted partner.
+
+NOTE: Disabling the CN check is not recommended for production environments.  |  false |
+| Host a| String |  Hostname for the proxy server. |  |
+| Port a| Number |  Port number for the proxy server. |  |
+| Username a| String |  Username to use for authenticating against the proxy server. |  |
+| Password a| String |  Password to use for authenticating against the proxy server. |  |
+| NTLM Domain a| String |  The domain to authenticate against the proxy. |  |
```

Comment:
description should be: The domain to use to authenticate against the proxy server